xcell/mappers/mapper_dummy.py

```diff
@@ -19,15 +19,18 @@ def __init__(self, config):
         self.l0 = self.config.get('l0', 20.)
         self.alpha = self.config.get('alpha', -1.)
         self.npix = hp.nside2npix(self.nside)
+        self.seed = None 
```

Comment:
I don't see any way of setting this seed, right? I.e. it can't be set in the config file or anywhere else.